src/components/header/ClaimWarningBanner.vue

```diff
@@ -1,6 +1,22 @@
 <template>
-  <div class="banner" :class="`banner--${network}`">
-    {{ network !== 2 ? $t('warning.claimRewards') : $t('warning.underDevelopmentShibuyaWarning') }}
+  <div>
+    <a
+      v-if="network !== 2"
```

Comment:
Please help change `2` to `endpointKey.SHIBUYA`. You can import the enum like this: `import { endpointKey,} from 'src/config/chainEndpoints';
` 
```suggestion
      v-if="network !== endpointKey.SHIBUYA"
```

Reply:
@impelcrypto fixed in [2b8bc39](https://github.com/AstarNetwork/astar-apps/pull/1144/commits/2b8bc39304f2cb4a469a423862f26f8c85b03a3b)